requirements.txt

```diff
@@ -1,4 +1,5 @@
 jinja2
+matplotlib
```

Comment:
I am not so sure how we should handle all the mapping-tester dependencies going forward.

Reply:
I would leave it as is right now, I guess most users would anyway have a dedicated venv for the dependencies here, then the difference is not such a big deal.